starfish/test/full_pipelines/cli/test_build.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shutil
+import subprocess
+import tempfile
+import unittest
+
+
+from starfish.util import clock
+
+
+class TestWithBuildData(unittest.TestCase):
+
+    STAGES = (
+        [
+            "starfish", "build",
+            "--fov-count=2", '--hybridization-dimensions={"z": 3}',
+            lambda tempdir: tempdir
+        ],
+        [
+            "validate-sptx", "--experiment-json",
+            lambda tempdir: os.sep.join([tempdir, "experiment.json"])
+        ],
+    )
+
+    def test_run_build(self):
+        tempdir = tempfile.mkdtemp()
```

Comment:
Can you replace this with `tempfile.TemporaryDirectory`?  The code you're copying from preceded the decision to support only python 3.

Reply:
Line removed in favor of `exec.stages`.